common/src/main/java/io/pravega/example/common/PravegaAppConfiguration.java

```diff
@@ -30,14 +30,22 @@
     private final StreamConfig inputStreamConfig;
     private final StreamConfig outputStreamConfig;
     private final boolean startAtTail;
+    private final boolean pravega_standalone;
+
+
 
     public PravegaAppConfiguration(String[] args) {
+
+//        URI controllerURI = URI.create(getEnvVar("PRAVEGA_CONTROLLER_URI", "tcp://10.247.115.55:9090"));
         URI controllerURI = URI.create(getEnvVar("PRAVEGA_CONTROLLER_URI", "tcp://localhost:9090"));
+        System.out.println("fkin controlleruri " + controllerURI);
         clientConfig = ClientConfig.builder().controllerURI(controllerURI).build();
-        defaultScope = getEnvVar("PRAVEGA_SCOPE", "examples");
+        defaultScope = getEnvVar("PRAVEGA_SCOPE", "video-demo");
```

Comment:
Revert this change. We should keep all Pravega examples in the "examples" scope to make it easier for users to test these examples.